posts/Yoann/2018-05-29-hibernate-search-5-10-1-Final-5-9-2-Final-5-6-5-Final.adoc

```diff
@@ -0,0 +1,57 @@
+= Triple bugfix release for Hibernate Search
+Yoann Rodière
+:awestruct-tags: [ "Hibernate Search", "Elasticsearch", "Releases" ]
+:awestruct-layout: blog-post
+---
+
+We just published bugfix releases for the branches currently in maintenance mode:
+
+Version 5.6.5.Final:: Latest version compatible with Hibernate ORM 5.0 and 5.1.
+Version 5.9.2.Final:: Latest version compatible with Hibernate ORM 5.2 (5.2.3.Final and later 5.2.x).
+Version 5.10.1.Final:: The latest and greatest, compatible with Hibernate ORM 5.3 and JPA 2.2.
```

Comment:
I would have use the reverse order for all the lists as 5.10.1 is definitely the most prominent release.

Reply:
Ha ha, I actually went back and forth several times, didn't know which order would be better :) Will do.